ionoscloud/resource_ipfailover.go

```diff
@@ -84,118 +100,137 @@ func resourceLanIPFailoverCreate(ctx context.Context, d *schema.ResourceData, me
 		return diags
 	}
 
-	d.SetId(*lan.Id)
-
-	return resourceLanIPFailoverRead(ctx, d, meta)
+	// Use the IP in order to generate the resource ID
+	d.SetId(uuid.NewV5(uuid.NewV5(uuid.NamespaceURL, "https://github.com/ionos-cloud/terraform-provider-ionoscloud"), ip).String())
```

Comment:
Better to define "https://github.com/ionos-cloud/terraform-provider-ionoscloud" as a variable for future purposes

Reply:
I think this can be a small function 
GenerateId(from string) string {
return uuid.NewV5(uuid.NewV5(uuid.NamespaceURL, "https://github.com/ionos-cloud/terraform-provider-ionoscloud"), from).String());
 }

Reply:
@cristiGuranIonos I agree

Reply:
@cristiGuranIonos done in ed1e170620e3d94cb90595dc3047a47258d4ae99